electrode-ota-server-dao-mariadb/test/ElectrodeOtaDaoRdms-spec.ts

```diff
@@ -1636,7 +1636,7 @@ describe("Data Access via RDBMS", function() {
             null
           );
         });
-        it("will return no release for matching appVersion against tagged release", () => {
+        it.skip("will return no release for matching appVersion against tagged release", () => {
```

Comment:
fix

Reply:
the signature of getNewestApplicablePackage() has changed, hence these 3 uses cases will be no more valid. maybe need to be updated as the current version works.

Reply:
updated the test as per signature of the getNewestApplicablePackage method